src/api/user/GetUserAnnotations.ts

```diff
@@ -0,0 +1,43 @@
+/* Import Dependencies */
+import axios from 'axios';
+
+/* Import Types */
+import { Annotation } from 'global/Types';
+
+/* Import API */
+import GetSpecimen from 'api/specimen/GetSpecimen';
+
+
+const GetUserAnnotations = async (token?: string) => {
+    if (token) {
+        let userAnnotations = <Annotation[]>[];
+
+        const endPoint = 'annotations/creator';
+
+        await axios({
+            method: "get",
+            url: endPoint,
+            responseType: 'json',
+            headers: {
+                'Authorization': `Bearer ${token}`,
+            },
+        }).then((result) => {
+            userAnnotations = result.data;
+
+            /* Temporary solution to include Specimen data */
```

Comment:
Add a backlog item for this

Reply:
Soulaine already had a look at this and said it was challenging, but will add it as a separate user story

Reply:
https://naturalis.atlassian.net/browse/DD-256?atlOrigin=eyJpIjoiZDlhZjliNGExNDkwNDBhYjhiODg0NTE4OTg5MzUwMmMiLCJwIjoiaiJ9

Reply:
Think it got a lot easier not we removed all the versions from the database. Should be easy now